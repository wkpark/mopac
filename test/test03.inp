 gnorm=0  & 
                      PM3 
 Methanol   Check that heat of formation is 51.8773 kcal/mol
   C
   O 1.4 1             1
   H 1.0 1 109 1       1 2
   H 1.0 1 109 1 120 1 1 2 3
   H 1.0 1 109 1 120 1 1 2 4
   H 1.0 1 120 1  30 1 2 1 3

