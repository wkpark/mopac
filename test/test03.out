 *******************************************************************************
 ** FRANK J. SEILER RES. LAB., U.S. AIR FORCE ACADEMY, COLO. SPGS., CO. 80840 **
 *******************************************************************************

                                 PM3 CALCULATION RESULTS


 *******************************************************************************
 *          MOPAC:  VERSION  6.00               CALC'D. 31-OCT-90               
 *   T=      - A TIME OF  3600.0 SECONDS REQUESTED
 *  DUMP=N   - RESTART FILE WRITTEN EVERY  3600.0 SECONDS
 *  PM3      - THE PM3 HAMILTONIAN TO BE USED
 *  GNORM=   - EXIT WHEN GRADIENT NORM DROPS BELOW .000E+00
 ***********************************************************************040BY040
 GNORM=0  &
                      PM3
 Methanol   Check that heat of formation is 51.8773 kcal/mol

    ATOM   CHEMICAL  BOND LENGTH    BOND ANGLE     TWIST ANGLE
   NUMBER  SYMBOL    (ANGSTROMS)     (DEGREES)      (DEGREES)
    (I)                  NA:I          NB:NA:I      NC:NB:NA:I     NA   NB   NC

      1      C       
      2      O         1.40000  *                                  1
      3      H         1.00000  *     109.00000  *                 1    2
      4      H         1.00000  *     109.00000  *  120.00000  *   1    2    3
      5      H         1.00000  *     109.00000  *  120.00000  *   1    2    4
      6      H         1.00000  *     120.00000  *   30.00000  *   2    1    3


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         C        0.0000    0.0000    0.0000
     2         O        1.4000    0.0000    0.0000
     3         H       -0.3256    0.9455    0.0000
     4         H       -0.3256   -0.4728   -0.8188
     5         H       -0.3256   -0.4728    0.8188
     6         H        1.9000    0.7500    0.4330
  H: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).                    
  C: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).                    
  O: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).                    


          RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =  7


            INTERATOMIC DISTANCES
0
                  C  1       O  2       H  3       H  4       H  5       H  6
 ------------------------------------------------------------------------------
     C    1   0.000000
     O    2   1.400000   0.000000
     H    3   1.000000   1.967636   0.000000
     H    4   1.000000   1.967636   1.637686   0.000000
     H    5   1.000000   1.967636   1.637686   1.637686   0.000000
     H    6   2.088061   1.000000   2.275716   2.831155   2.568494   0.000000

  GNORM HAS BEEN SET TOO LOW, RESET TO 0.01
 CYCLE:   1 TIME:   1.18 TIME LEFT:   3597.6 GRAD.:    30.572 HEAT:-50.279019    
 CYCLE:   2 TIME:   1.13 TIME LEFT:   3596.5 GRAD.:     9.672 HEAT:-51.305837    
 CYCLE:   3 TIME:   0.65 TIME LEFT:   3595.8 GRAD.:     8.352 HEAT:-51.336586    
 CYCLE:   4 TIME:   0.67 TIME LEFT:   3595.2 GRAD.:     2.674 HEAT:-51.715802    
 CYCLE:   5 TIME:   0.90 TIME LEFT:   3594.3 GRAD.:     5.474 HEAT:-51.825893    
 CYCLE:   6 TIME:   0.59 TIME LEFT:   3593.7 GRAD.:     2.996 HEAT:-51.862820    
 CYCLE:   7 TIME:   0.58 TIME LEFT:   3593.1 GRAD.:     0.837 HEAT:-51.875435    
 CYCLE:   8 TIME:   0.54 TIME LEFT:   3592.5 GRAD.:     0.353 HEAT:-51.876568    
 CYCLE:   9 TIME:   0.54 TIME LEFT:   3592.0 GRAD.:     0.216 HEAT:-51.877032    
 CYCLE:  10 TIME:   0.58 TIME LEFT:   3591.4 GRAD.:     0.104 HEAT:-51.877258    
 CYCLE:  11 TIME:   0.53 TIME LEFT:   3590.9 GRAD.:     0.053 HEAT:-51.877301    
 CYCLE:  12 TIME:   0.74 TIME LEFT:   3590.2 GRAD.:     0.012 HEAT:-51.877323    
 TEST ON GRADIENT SATISFIED
 PETERS TEST SATISFIED 

 -------------------------------------------------------------------------------
 GNORM=0  &
                      PM3
 Methanol   Check that heat of formation is 51.8773 kcal/mol


     PETERS TEST WAS SATISFIED IN BFGS OPTIMIZATION           
     SCF FIELD WAS ACHIEVED                                   


                                PM3    CALCULATION
                                                       VERSION  6.00
                                                       31-OCT-90               




          FINAL HEAT OF FORMATION =        -51.87733 KCAL


          TOTAL ENERGY            =       -474.14524 EV
          ELECTRONIC ENERGY       =      -1047.64242 EV
          CORE-CORE REPULSION     =        573.49717 EV

          IONIZATION POTENTIAL    =         11.13846
          NO. OF FILLED LEVELS    =          7
          MOLECULAR WEIGHT        =     32.042


          SCF CALCULATIONS  =               19
          COMPUTATION TIME =  10.730 SECONDS





    ATOM   CHEMICAL  BOND LENGTH    BOND ANGLE     TWIST ANGLE
   NUMBER  SYMBOL    (ANGSTROMS)     (DEGREES)      (DEGREES)
    (I)                  NA:I          NB:NA:I      NC:NB:NA:I     NA   NB   NC

      1      C       
      2      O         1.39481  *                                  1
      3      H         1.09703  *     112.16759  *                 1    2
      4      H         1.09360  *     104.47406  *  118.43394  *   1    2    3
      5      H         1.09702  *     112.15384  *  118.43044  *   1    2    4
      6      H         0.94878  *     107.45548  *   61.45274  *   2    1    3


            INTERATOMIC DISTANCES
0
                  C  1       O  2       H  3       H  4       H  5       H  6
 ------------------------------------------------------------------------------
     C    1   0.000000
     O    2   1.394815   0.000000
     H    3   1.097026   2.074530   0.000000
     H    4   1.093599   1.975847   1.788179   0.000000
     H    5   1.097018   2.074360   1.786875   1.788206   0.000000
     H    6   1.907776   0.948776   2.313989   2.769553   2.315193   0.000000


                  EIGENVALUES

 -38.18274 -26.49461 -18.00425 -15.52389 -15.51750 -12.46136 -11.13846   3.50820
   3.89413   4.22633   4.38969   5.64675


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

         ATOM NO.   TYPE          CHARGE        ATOM  ELECTRON DENSITY
           1          C           0.0699          3.9301
           2          O          -0.3088          6.3088
           3          H           0.0085          0.9915
           4          H           0.0409          0.9591
           5          H           0.0085          0.9915
           6          H           0.1809          0.8191
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.    -0.697     0.296     0.543     0.931
 HYBRID        -0.149     0.289     0.532     0.623
 SUM           -0.846     0.585     1.074     1.487


          CARTESIAN COORDINATES 

    NO.       ATOM               X         Y         Z

     1         C                  0.0000    0.0000    0.0000
     2         O                  1.3948    0.0000    0.0000
     3         H                 -0.4139    1.0159    0.0000
     4         H                 -0.2733   -0.5042   -0.9312
     5         H                 -0.4137   -0.5554    0.8508
     6         H                  1.6794    0.4325    0.7950


          ATOMIC ORBITAL ELECTRON POPULATIONS

   1.12401   0.83957   0.97502   0.99154   1.80692   1.19362   1.83048   1.47772
   0.99148   0.95907   0.99146   0.81908
  CHANNEL 12 OPENED



 TOTAL CPU TIME:            10.97 SECONDS

 == MOPAC DONE ==
